### PR TITLE
Update PartsAnalyses to 3.0.0

### LINKED ISSUE #513: [Model Update]: io.catenax.parts_analyses:2.0.0 to 3.0.0
## Update Reason
Use quality decided to put many of the current properties to optional for CX 24.05 release. In addition, some typos in the model example payload and in the description exists.
For model io.catenax.parts_analyses:2.0.0 I assume that some properties are now set to optional - but I don't expect significant change in structure

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists - Catena-X quality use case
- [x] The potential updates are discussed with all stakeholders - discussed in Catena-X use case quality
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [X] The potential update will be either 
  - [X] backward compatible or 
  - [X] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by TorstenHirsch**
## Description
Implementing CX use quality decision to set most of the properties to optional.
### Added
- metaInformation property added to root entity
- integration of the shared UUID characteristic and RegEx for the vehicle Catena-X identifie
- introducing new characteristic for unique identifiers
- introduced new prarent properties that could be used for Tier-n collaboration

### Changed
- changed all properties to optional except one unique identifier per entity

### Removed
- oem properties -> replaced by parent properties

 -->

Closes #513 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by TorstenHirsch** [comment]
@agg3fe modelling_team

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+:status a samm:Property ;
+   samm:preferredName "Status"@en ;
+   samm:description "Status of this part analysis"@en ;
+   samm:characteristic :StatusCharacteristic ;
+   samm:exampleValue "new" .
+
+:addtionalInformation a samm:Property ;
+   samm:preferredName "Additional information"@en ;
+   samm:description "You can use this key:value list for additional properties that were not defined in this aspect model."@en ;
+   samm:characteristic :Text .
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:155]: why not to use` samm-c:Text` ?

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+   samm:characteristic :Keys ;
+   samm:exampleValue "Steel quality" .
+
+:values a samm:Property ;
+   samm:preferredName "Values"@en ;
+   samm:description "Values for the keys defined"@en ;
+   samm:characteristic :Values ;
+   samm:exampleValue "Stainless steel" .
+
+:Keys a samm-c:List ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:214]: missing `preferredName` and `description`

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+   samm:dataType xsd:string .
+
+:Values a samm-c:List ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:217]: same here

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+:selectionStart a samm:Property ;
+   samm:preferredName "Selection start timestamp"@en ;
+   samm:description "Describes the starting point for providng records in this data set."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2023-01-01T00:00:00" .
+
+:selectionEnd a samm:Property ;
+   samm:preferredName "Selection end timestamp"@en ;
+   samm:description "Describes the end point for providng records in this data set."@en ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:171]: typo `providng`

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+:selectionCriteria a samm:Property ;
+   samm:preferredName "Selection criteria"@en ;
+   samm:description "Describes the selection criteria a Catena-X data provider has applied to create this data set."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Data set is the result of all Claims where Claim.repairDate is in the range from selectionStartTimestamp to selectionEndTimestamp, supplier is Smith Corporation and supplier component is accelaration sensor" .
+
+:selectionStart a samm:Property ;
+   samm:preferredName "Selection start timestamp"@en ;
+   samm:description "Describes the starting point for providng records in this data set."@en ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:165]: typo `providng`

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+:catenaXQualityTaskId a samm:Property ;
+   samm:preferredName "Quality Task ID"@en ;
+   samm:description "The fully anonymous and unique Catena-X ID of this quality task. This id is unique inside Catena-X dataspace."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "430f56d3-1234-1234-1234-abc123456789" .
+
+:isDefect a samm:Property ;
+   samm:preferredName "Part defect flag"@en ;
+   samm:description "True: Analysis turned out that analyzed part is defect accroding part's specification."@en ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:94]: typo accroding

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+   samm:characteristic :MetaCharacteristic .
+
+:ListOfPartsAnalyses a samm-c:List ;
+   samm:preferredName "Part Analyses"@en ;
+   samm:description "A list for multiple part analyses"@en ;
+   samm:dataType :PartAnalysis .
+
+:MetaCharacteristic a samm:Characteristic ;
+   samm:preferredName "MetaCharacteristic"@en ;
+   samm:description "Chracteristic for the meta information."@en ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:55]: typo `Chracteristic`

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+   samm:preferredName "Text"@en ;
+   samm:description "Describes a Property which contains plain text. This is intended exclusively for human readable strings, not for identifiers, measurement values, etc."@en ;
+   samm:dataType :AdditionalInformation .
+
+:AdditionalInformation a samm:Entity ;
+   samm:preferredName "Additional information"@en ;
+   samm:description "Additional information is stored as list of keys and list of values"@en ;
+   samm:properties ( :keys :values ) .
+
+:keys a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:202]: Property and the referenced Characteristic should not have the same name

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+   samm:preferredName "Keys"@en ;
+   samm:description "a list of keys"@en ;
+   samm:characteristic :Keys ;
+   samm:exampleValue "Steel quality" .
+
+:values a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:208]: Property and the referenced Characteristic should not have the same name

**Review by agg3fe — CHANGES_REQUESTED** [review]
Please check my comments

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 2 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed/restructered

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 4 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 5 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 6 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 8 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl (hunk 9 above)
Comment (TorstenHirsch) [reply io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl]: fixed

**Comment by TorstenHirsch** [comment]
@agg3fe Please recheck - can you than give MS2 status
modelling_team

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+   samm:description "This entity groups all meta information properties. Meta information properties give hints for a Catena-X data consumer to better understand, how this data set was created on the Catena-X data provider side."@en ;
+   samm:properties ( :selectionCriteria [ samm:property :selectionStart; samm:optional true ] [ samm:property :selectionEnd; samm:optional true ] ) .
+
+:recordStatus a samm:Property ;
+   samm:preferredName "Record status"@en ;
+   samm:description "The record operation enunumeration can be used to realize delta update concept.\nDelta update concept means:\n- You transfer an initial load of data\n- After the first week only the delta to the initial load is transferred\n\n\nThe record status describes whether this record is:\n- new=>This record is transferred the first time\n- update=> Some properties of this record have changed compared to a previous transfer\n- delete=> This record was transferred in the initial load or in a previous delta update, but is not \nused any more and therefore it should be deleted on data consumer side\n- same=> This record was transferred in the initial load or in a previous delta update"@en ;
```
Comment (agg3fe) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:50]: please fix typo `enunumeration`. should be enumeration.

**Comment by TorstenHirsch** [comment]
@agg3fe 
modelling_team

**Review by agg3fe — APPROVED** [review]
LGTM

File: io.catenax.parts_analyses/RELEASE_NOTES.md [hunk 11]
```diff
@@ -2,9 +2,22 @@
...
+- introduced new prarent properties that could be used for Tier-n collaboration
+
+### Changed
+- changed all properties to optional except one unique identifier per entity
 
+### Removed
+- oem properties -> replaced by parent properties
 
-## [2.0.0] - 2023-06-23
+## [2.0.0] - 2023-05-09
```
Comment (agg3fe) [inline io.catenax.parts_analyses/RELEASE_NOTES.md:20]: please check this.

File: io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "Steel quality" .
+
+:value a samm:Property ;
+   samm:preferredName "Value"@en ;
+   samm:description "The value for this additional information that refers to the linked unique key."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Stainless steel" .
+
+:AdditionalInformationCharacteristic a samm:Characteristic ;
```
Comment (catroest) [inline io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl:213]: samm:preferredName and samm:description missing

**Pull request merged**